utils/isOwner.js

```diff
@@ -0,0 +1,15 @@
+const isOwner = async (collection, userId, id) => {
+  try {
+    const singleCollection = await collection.findOne({ id });
+
+    if(userId === singleCollection.user.id){
```

Comment:
What if record can't be found? I think we should check singleCollection is not null first. Beyond that, does ever collection have a user.id field?

Reply:
That's true I forgot to check  if the record can't be found.  There are some collections that don't have a user id field, but I figured this function would only be used on collections that do.

Reply:
Is the try catch going to prevent crashing the app if the collection doesn't have user id field?

Reply:
That I have not tested. Do you think I should add a check that checks if the collection doesn't have a user id? My thought process was that this function will only ever be used on collections that do have user id fields, but if you think I should add it I'll add it.

Reply:
It's not always safe to assume that the caller will pass in good inputs.

Reply:
got it